sources/Java.java

```diff
@@ -0,0 +1,41 @@
+import java.util.*;
+import java.io.*;
+import java.text.DecimalFormat;
+public class Java {
+
+	public static void main(String[] args) {
+		int range = Integer.parseInt(args[0]);
+		int trails = Integer.parseInt(args[1]);		
+		prng(range, trails);
+	}
+	static void prng(int numbers, int trails) {
+		int frequency[] = new int[numbers];
+		double probability[] = new double[numbers];
+		int start_time = (int) System.currentTimeMillis();
+		Random random = new Random();
+		for(int i=0;i<trails;i++) {
+			int r_number = random.nextInt(numbers);
+			frequency[r_number] +=1;
```

Comment:
Source formatting to be consistent.  In some places we have spaces after `=` or `)`.